frame/offences/src/migration.rs

```diff
@@ -0,0 +1,122 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Config;
+use frame_support::{storage::StorageValue, traits::Get, weights::Weight};
+use sp_staking::offence::OnOffenceHandler;
+
+mod deprecated {
+    use crate::{Config, OffenceDetails, Perbill, SessionIndex};
```

Comment:
spaces 😱

Reply:
![](https://media.tenor.com/images/3b5c3ed48f63b05c716972bfc8c7e526/tenor.gif)

Reply:
Good catch! we must have a CI check of this :)